backend/internal/application/store/constants.go

```diff
@@ -30,40 +30,42 @@ var (
 
 	// QueryCreateApplication is the query to create a new application with basic details.
 	QueryCreateApplication = dbmodel.DBQuery{
-		ID:    "ASQ-APP_MGT-01",
-		Query: "INSERT INTO SP_APP (APP_ID, APP_NAME, DESCRIPTION, AUTH_FLOW_GRAPH_ID) VALUES ($1, $2, $3, $4)",
+		ID: "ASQ-APP_MGT-01",
+		Query: "INSERT INTO SP_APP (APP_ID, APP_NAME, DESCRIPTION, AUTH_FLOW_GRAPH_ID, REGISTRATION_FLOW_GRAPH_ID) " +
+			"VALUES ($1, $2, $3, $4, $5)",
 	}
 	// QueryCreateOAuthApplication is the query to create a new OAuth application.
 	QueryCreateOAuthApplication = dbmodel.DBQuery{
 		ID: "ASQ-APP_MGT-02",
-		Query: "INSERT INTO IDN_OAUTH_CONSUMER_APPS (APP_ID, CONSUMER_KEY, CONSUMER_SECRET, CALLBACK_URIS, GRANT_TYPES) " +
-			"VALUES ($1, $2, $3, $4, $5)",
+		Query: "INSERT INTO IDN_OAUTH_CONSUMER_APPS (APP_ID, CONSUMER_KEY, CONSUMER_SECRET, CALLBACK_URIS, " +
```

Comment:
There are no changes here.. Shall we avoid this diff, if its not reqiured.

Reply:
This exceeds 120 max line lenngth.
https://github.com/asgardeo/thunder/blob/86311baab0e1aaa3d5b3b3dc1df25ee47fed63a7/backend/.golangci.yml#L81

Reply:
Any idea why this wasn't caught earlier by the linter?